lib/index.js

```diff
@@ -1 +1,3 @@
 require ('./css/styles');
+
+const firebase = require('./firebase');
```

Comment:
Yeah that line is for using firebase to potentially create functions using it in the future. Other than me creating stuff that wouldn't really be relevant to our app I don't think there's a way to test it. Once we have an 'Add Contact' page we can test that those inputs get pushed into the server. Should I remove that line for now?